server/client_server.go

```diff
@@ -22,21 +23,126 @@ func newServerClient(jiraClient *jira.Client) Client {
 	}
 }
 
+type IssueInfo struct {
+	Values []*jira.MetaIssueType `json:"values,omitempty"`
+}
+
+type FieldInfo struct {
+	Values []interface{} `json:"values,omitempty"`
+}
+
+type FieldValues struct {
+	Id string `json:"fieldId,omitempty"`
+}
+
+type FieldId struct {
+	Values []FieldValues `json:"values,omitempty"`
+}
+type Version struct {
+	VersionInfo string `json:"version,omitempty"`
+}
+
 // GetCreateMeta returns the metadata needed to implement the UI and validation of
 // creating new Jira issues.
 func (client jiraServerClient) GetCreateMeta(options *jira.GetQueryOptions) (*jira.CreateMetaInfo, error) {
-	cimd, resp, err := client.Jira.Issue.GetCreateMetaWithOptions(options)
+	apiEndpoint := "rest/api/2/serverInfo"
```

Comment:
Here i think there will a file where all the endpoint constants will be initialised put this constant there 
```suggestion
	ServerInfoApiEndpoint := "rest/api/2/serverInfo"
```